starter-code/apple-pie-starter-code/style.css

```diff
@@ -0,0 +1,37 @@
+
+h1, body, html, main {
+  margin: 0;
+}

```

Comment:
Reset ✅ 
But try to use the one that we have on the [cheat-sheet](https://github.com/ironhack/bcn-webdev-cheatsheet/blob/master/m1/css/boilerplate.css)